templates/support/esm.html

```diff
@@ -17,29 +17,43 @@
       <img src="{{ ASSET_SERVER_URL }}9e59756d-shield-ESM.svg" width="150" alt="" />
     </div>
   </div>
-</div>
+</section>
 
 {% include "shared/_call-sales.html" %}
 
-<div class="p-strip is-deep is-bordered">
+<section class="p-strip is-deep is-bordered">
   <div class="row">
-    <div class="col-8 suffix-1">
+    <div class="col-8">
       <h2>Benefits of ESM</h2>
       <p>All Ubuntu 12.04 LTS users are encouraged to upgrade to Ubuntu 16.04 LTS. But for those who cannot upgrade immediately, Ubuntu 12.04 ESM updates will help ensure the on-going security and integrity of their LTS systems.</p>
       <p>Canonical&rsquo;s Ubuntu security team provide fixes on high and critical CVEs (common vulnerabilities and exposures) for the most commonly used server packages in the Ubuntu main archive. ESM provides the essential continuation of the security updates that Ubuntu 12.04 LTS users have always received via a secure, private archive.</p>
-      <p><a href="https://wiki.ubuntu.com/SecurityTeam/ESM/12.04" class="p-link--external">Learn which server packages are maintained</a></p>
     </div>
-    <div class="col-4 p-card--highlighted">
-      <h3 class="p-card__title">Ubuntu 12.04 ESM FAQ</h3>
-      <p class="p-card__content">Answers to your most frequently asked questions about Ubuntu 12.04 ESM.</p>
-      <footer class="p-card__footer">
-        <a href="https://insights.ubuntu.com/2017/04/11/faqs-ensuring-the-ongoing-security-compliance-of-ubuntu/" title="Read the Ubuntu 12.04 ESM FAQs" class="p-link--external">Read the FAQs</a>
-      </footer>
+  </div>
+</section>
+
+<section class="p-strip is-deep is-bordered">
+  <div class="row">
+    <div class="col-10">
+      <h2>Resources</h2>
+    </div>
+  </div>
+  <div class="row p-divider u-equal-height u-vertically-spaced">
+    <div class="col-4 p-divider__block">
+      <h4><a href="https://pages.ubuntu.com/Cloud_CS_ITstrategen.html" class="p-link--external">ITstrategen case study</a></h4>
```

Comment:
Should the headings be h3s with p-heading--four?

Reply:
Yeah, you're right